samza-api/src/main/java/org/apache/samza/operators/MessageStream.java

```diff
@@ -153,6 +153,7 @@
    * The merged {@link MessageStream} contains messages from all {@code streams} in the order they arrive.
    *
    * @param streams {@link MessageStream}s to be merged
+   * @param <T> the common type of messages in the streams
```

Comment:
Minor:  

`s/<T> the common type of messages in the streams/the type of messages in each of the streams/`